frontend/components/item/value/type/Text.vue

```diff
@@ -1,8 +1,7 @@
 <template>
   <div>
     <span v-if="!isUserLogged">
-      <!-- eslint-disable-next-line vue/no-v-html -->
```

Comment:
Please enable _eslint_ plugin in you local development and recover this line to avoid a _eslint_ warning.